rclcpp/src/rclcpp/parameter_client.cpp

```diff
@@ -152,7 +152,7 @@ AsyncParametersClient::get_parameters(
       auto & pvalues = cb_f.get()->values;
 
       for (auto & pvalue : pvalues) {
-        auto i = &pvalue - &pvalues[0];
+        size_t i = static_cast<size_t>(&pvalue - &pvalues[0]);
```

Comment:
With the explicit static case you should be able to continue to use `auto` - no need to repeat the type.

Reply:
sure, done